resources/assets/components/Tags/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { map } from 'lodash';
+
+class Tags extends React.Component {
+  render() {
+
+    return (
+        <ul className="form-actions -inline">
+          <li><b> Tags </b></li>
```

Comment:
Are there any issues listed in the console? One thought is to make sure to wrap those in a `<div>` when returning from the `render` method if you're not. React doesn't like when you return multiple elements for a component.